syntax = "proto3";

package dataflow;

message TradeEvent {
  string trade_id = 1;
  string account_id = 2;
  string business_unit = 3;
  double amount = 4;
  string symbol = 5;
  int64 traded_at = 6;
}